src/modules/teams/containers/TeamsPage.js

```diff
@@ -0,0 +1,99 @@
+import { h, Component } from "preact";
+import { connect } from "preact-redux";
+import { Page } from "src/components/Page";
+import { Team } from "../components/Team";
+
+import styles from "./TeamsPage.scss";
+
+import { fetchAllTeams, replyToInvite, selectTeam } from "../actions";
+import { loadTeams } from "../../../localStorage";
+
+import { route } from "preact-router";
+import I18n from "src/config/i18n";
+
+export class TeamsPage extends Component {
+
+    constructor(props) {
+        super(props);
+        this.props.fetchAllTeams(this.props.user.apiToken);
```

Comment:
Fetching data from the backend can best be done in `componentDidMount`. You want to have your initial state in the constructor telling users there is no data yet when it's undefined until the component is fully mounted